doc/manual/manual.tex

```diff
@@ -6656,7 +6656,71 @@ \subsubsection{Crustal deformation}
+
+As noted above, the mechanically weak seed placed within the mantle localizes the majority deformation onto two conjugate shear bands that propogate from the surface of the seed to the free surface.  After 5 million years of extension background 'stretching' is clearly visible in the strain-rate field, but deformation is still largely focused within the set of conjugate shear bands originating at the weak seed  (\ref{fig:continental_extension_cookbook_strainrate_5e6yr}).  As expected, crustal thickness and surface topography patterns reveal a relatively symmetric horst and grabben structure, which arises from displacements along the shear bands (\ref{fig:continental_extension_cookbook_upper_5e6yr}).  
+
+With continued extension, further crustal thinning and surface topography development would likely occur in response to displacement along the conjugate shear bands. However, given that the model only extends to 100 km depth, the simulation will not produce a realistic representation of continental breakup due to the lack of an upwelling asthenospheric layer.  Indeed, numerical studies that examine continental breakup, rather than just the initial stages of continental extension, inlcude an asthenospheric layer or modified basal boundary conditions (e.g. winkler boundary condition \cite[for example]{Bru14})
+as temperature variations associated with lithospheric thinning exert a first-order influence on the deformation patterns.
+
+On that note, it is important to consider that the non-linearity of visco-plastic rheologies and mesh-dependence of brittle shear bands make lithospheric deformation models highly sensitive to a large number of parameters.  In order to ensure the conclusions drawn from a series of numerical experiments are robust, one should complete a sensitivity test for a large range of parameters including grid resolution, model geometry, boundary condtions, initial composition and temperature conditions, material properties, composition discretization, CFL number and solver settings.  If you are new to modeling lithospheric processes, a reasonable starting point is to try and reproduce results from a relevant previous study and then perform a sensitivity test for the parameters listed above.  While highly time consuming, completing this procedure will prove invaluable when you design and assess the results of your own numerical study.
+ 
+\begin{figure}
+\centering
+\includegraphics[width=\textwidth]{cookbooks/continental_extension/continental_extension_cookbook_strainrate_5e6yr.png}
+\caption{\it Strain rate ($s^{-1}$) after 5e6 years of extension.} 
+\label{fig:continental_extension_cookbook_strainrate_5e6yr}
+\includegraphics[width=\textwidth]{cookbooks/continental_extension/continental_extension_cookbook_upper_5e6yr.png}
+\caption{\it Value of upper crust compositional field after 5e6 years of extension.} 
+\label{fig:continental_extension_cookbook_upper_5e6yr}
```

Comment:
can you have two `\caption` and two `\label` parts in one figure environment? If not, just make it two environments.